src/pages/howItWorks/Howitwork2.jsx

```diff
@@ -17,7 +17,7 @@ export default function Howitwork2() {
       </div>
 
       <section className=" pb-32 max-md:py-20 max-sm:py-20 ">
-        <div className=" grid grid-cols-2 gap-5 px-24 max-md:flex flex-col max-md:px-8 ">
+        <div className=" grid grid-cols-2 gap-5 px-24 max-sm:flex-col max-md:flex max-md:flex-col max-md:px-8 ">
```

Comment:
- [ ] Good job so far 👍🏽  but I noticed your cards are not responsive on mobile. Since the image is to be stacked on the mobile, i will suggest you change the `grid-cols-2` to `grid-cols-1` and also adjust the padding around the card to match the design 👍🏽  . 

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/203511828-844733c4-8407-45ba-9e9b-4c581199be81.png)

FIgma Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/203512188-83d53208-0cf6-4bdd-98a7-25649e5fd561.png)